frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -149,7 +149,7 @@ function FileManagerMenu:setUpdateItemTable()
 
     -- setting tab
     self.menu_items.filebrowser_settings = {
-        text = _("Settings"),
+        text = _("File Management"),
```

Comment:
Settings is much clearer. Settings is something you ignore unless you're looking for settings. File management implies a great many things, all of which are incorrect.

Reply:
my issue lies with many places using "Settings" (not like "x settings") but literally just "settings", I would like to reduce it to just one. I am happy to keep it as "settings" but the "settings" elsewhere would eventually go away.